layers/transportation/transportation.sql

```diff
@@ -359,9 +359,17 @@ FROM (
          FROM osm_highway_linestring
          WHERE NOT is_area
            AND
-               CASE WHEN zoom_level = 12 THEN transportation_filter_z12(highway, construction)
+               CASE WHEN zoom_level = 12 THEN
+                         CASE WHEN transportation_filter_z12(highway, construction) THEN TRUE
+                              WHEN network IN ('hike-international', 'hike-national', 'hike-regional') THEN TRUE
```

Comment:
Shouldn't this condition be same as in `transportation_name` layer?
https://github.com/openmaptiles/openmaptiles/blob/c4d2c80801688ba91095af6b941ead021e171148/layers/transportation_name/transportation_name.sql#L117
 
```suggestion
                              WHEN route_rank = 1 THEN TRUE
```